sync/syncer.go

```diff
@@ -339,7 +339,10 @@ func (s *Syncer) handleWeaklySynced() {
 	}
 
 	//validate current layer if more than s.ValidationDelta has passed
-	s.handleCurrentLayer()
+	if err := s.handleCurrentLayer(); err != nil {
+		s.With().Error("Node is out of synce", log.Err(err))
+		return
+	}
```

Comment:
shouldn't we change the gossip status?

Reply:
we should leave it the same as it was, 
the status is "pending" i can add it explicitly if you think it's clearer.

Reply:
Are you sure? where in this flow do we set it to pending?

Reply:
fixed